cirq-core/cirq/ops/common_channels.py

```diff
@@ -246,7 +246,7 @@ def asymmetric_depolarize(
 
 
 @value.value_equality
-class DepolarizingChannel(gate_features.SupportsOnEachGate, raw_types.Gate):
+class DepolarizingChannel(raw_types.Gate):
```

Comment:
Question for my understanding: `isinstance(cirq.DepolarizingChannel(...), SupportsOnEachGate)` used to return True, but now it will return False, right? Or am I misunderstanding the metaclass business?

Reply:
That's correct, I missed that and will fix it.